doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -103,7 +103,7 @@ Hot Reload features vary between platforms and IDE, you can check below the list
         </tr>
         <tr>
             <td>WinAppSDK</td>
-            <td>✅<sup><a href="#hr-footnotes">[5]</a></sup>🐞</td><td>✅🐞</td>
+            <td>✅🐞<sup><a href="#hr-footnotes">[5]</a></sup></td><td>✅🐞</td>
```

Comment:
The order was the right one, the node is for the hot reload not the debugger. (other parts are like this as well). We can keep it like this if it is confusing, though.

Reply:
In my head when I see this visually, it means the footnote is only a mention for "without debugger". Maybe it is just me but I think it could add confusion.
![image](https://github.com/user-attachments/assets/aa7077d0-d500-4313-a519-62be8ef252e9)